intel_extension_for_transformers/llm/runtime/graph/README.md

```diff
@@ -115,7 +115,40 @@ LLM running script args explanations:
 | --color           | colorise output to distinguish prompt and user input from generations   |
 | --keep            | number of tokens to keep from the initial prompt (default: 0, -1 = all) |
 
+### 4. Run Models of Python Interface
 
-### 4. Tensor Parallelism cross nodes/sockets
+Here is how to install itrex_llm_runtime from source.
+```bash
+git submodule update --recursive --init
+python setup.py install
+```
+
+```python
+import itrex_llm_runtime.mpt_cpp as cpp_model
```

Comment:
@a32543254 is there any better python api/package?

Reply:
we are working on this interface:

```
from intel_extension_for_transformers.transformers import AutoModel
model_name = "mosaicml/mpt-7b"
model = AutoModel.from_pretrained(model_name, use_cpp_graph=True)
model.gernerate(prompt = prompt, streamer)
```